dGame/Entity.cpp

```diff
@@ -2197,9 +2197,3 @@ void Entity::SetRespawnRot(const NiQuaternion& rotation) {
 	auto* characterComponent = GetComponent<CharacterComponent>();
 	if (characterComponent) characterComponent->SetRespawnRot(rotation);
 }
-
-void Entity::SetScale(const float scale) {
-	if (scale == m_Scale) return;
-	m_Scale = scale;
-	Game::entityManager->SerializeEntity(this);
-}
```

Comment:
Not really an issue, but I think your text editor isn't set to end files with a newline.

Reply:
I switch between python and C++ a bunch, so I leave autoformatting off unless I'm specifically doing formatting passes. newlines at eof's are something I miss when coding manually